flink-runtime/src/main/java/org/apache/flink/runtime/operators/coordination/OperatorCoordinator.java

```diff
@@ -122,7 +123,18 @@
 	 * instead. Any exception propagating from this method may be treated as a fatal error for the
 	 * JobManager, crashing the JobManager, and leading to an expensive "master failover" procedure.
 	 */
-	void checkpointComplete(long checkpointId);
+	@Override
+	void notifyCheckpointComplete(long checkpointId);
+
+	/**
+	 * The OperatorCoordinator should not throw exception when notified an aborted checkpoint.
+	 * Instead, the implementation should just fail the task by using
+	 * {@link Context#failJob(Throwable)}.
+	 *
+	 * @param checkpointId The ID of the checkpoint that has been aborted.
+	 */
+	@Override
+	default void notifyCheckpointAborted(long checkpointId) {}
```

Comment:
Same as above - not sure I understand why these methods are being overridden here.